lib/internal/perf/performance_entry.js

```diff
@@ -31,10 +31,12 @@ function isPerformanceEntry(obj) {
 }
 
 class PerformanceEntry {
-  constructor(skipThrowSymbol = undefined) {
+  constructor(skipThrowSymbol = undefined, name = undefined, type = undefined, start = undefined, duration = undefined) {
```

Comment:
Why do you even set the parameters by default to undefined?

Reply:
Without `undefined`, the `idlharness` tests break because `PerformanceEntry` is supposed to not have constructor parameters._